landkreise/helper.py

```diff
@@ -2,8 +2,16 @@
 import datetime
 import locale
 
+# NOTE: these have to be sorted, with the most general regex at the bottom!
```

Comment:
I really like the idea with rege identifying the date format 
That looks like a good approach! In my branch is a function_generator that returns a function that can be given multiple date formats and then just blindly tries them all. It also allows to pass in multiple locales , but restricting the attempt by a map with key:regex => value:dateformat looks like a good approch. The regex could also be used to identify the locale (if there are clear indications). The problem is that a central date format map might not work once you encounter American date formats Month and Day swapped and it can be really difficult to identify.